src/main/resources/db/migration/changes/007-create-Instructors.json

```
@@ -0,0 +1,34 @@
+{
+  "databaseChangeLog": [
+    {
+      "changeSet": {
+        "id": "Instructor-1",
+        "author": "shawnz",
```

Please make this small change so that our naming conventions are consistent.  You can just click "commit suggestion" to do that.

```suggestion
        "id": "007-create-Instructors,
        "author": "yxz29315",
```